client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -112,7 +114,17 @@ public static StreamManager create(ClientConfig clientConfig) {
      * @return True if scope is deleted
      */
     boolean deleteScope(String scopeName);
-    
+
+    /**
+     * Get information about a given Stream, {@link StreamInfo}.
```

Comment:
Can we expand this javadoc to say what is the info that it returns?

Reply:
updated the javadoc.